docs/source/working_with_aiida/groups.rst

```diff
@@ -2,22 +2,23 @@ Groups
 ------
 
 Groups provide an additional level of data organization based on some common
-property(ies) shared between them. As by default AiiDA manages only casual
+property(ies) shared between data units. As by default AiiDA manages only casual
 relationships between calculations and data objecs -- it is often desired to
-organize them in Groups.
+organize them in groups.
 
-Nodes of any types may be organized in Groups. Unlike Nodes, Groups can be
-modified at any time. Here we profide the list of typical operations that may
-be performed with Groups:
-
-.. note:: Any deletion operation like delition of a Group or nodes from a Group
-  will not delete the nodes themselfs. They will remain in you AiiDA database.
+Nodes of any types may be organized in groups. Unlike nodes, groups can be
+modified at any time. Here we profide a list of typical operations that may
+be performed with groups:
 
+.. note:: Any deletion operation related to groups won't affect the nodes themselfs.
+  For example if you delete a group, the nodes that were belonging to the group will remain
+  in the database. Same happens if you delete nodes from the group -- they will remain
+  in the database but won't belong to the group anymore.
 
 1. **Create a new Group.**
     From command line interface::
```

Comment:
From command line interface -> From the command line interface

Please apply these to all the other occurrences as well

Reply:
done